frontend/src/app/components/dashboard/db-table-view/saved-filters-panel/saved-filters-dialog/saved-filters-dialog.component.html

```
@@ -49,170 +52,190 @@ <h1 mat-dialog-title class="filters-header">
-                            </mat-option>
-                        </mat-select>
-                    </mat-form-field>
+                        <mat-form-field *ngIf="getComparatorType(getInputType(value.key)) === 'number'"
+                            appearance="outline"
+                            class="comparator-select-field">
+                            <mat-select name="numberComparator-{{value.key}}"
+                                [(ngModel)]="tableRowFieldsComparator[value.key]"
+                                (ngModelChange)="updateComparator($event, value.key)">
+                                <mat-select-trigger>
+                                    <span class="comparator-trigger__symbol">
+                                        {{ {eq: '=', gt: '>', lt: '<', gte: '≥', lte: '≤'}[tableRowFieldsComparator[value.key]] || tableRowFieldsComparator[value.key] }}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)